deployment/utils.py

```diff
@@ -43,6 +43,11 @@ def load_tags():
     if 'owner' not in tags:
         tags['owner'] = os.popen("git config --get user.email").read().rstrip()
 
+    for k, v in tags.items():
+        if v is None:
+            raise RuntimeError(
+                "None is not legal, in tag {!r}".format(k))
```

Comment:
Minor formatting change: if an opening parenthesis is on its own line then so should be the closing one.

Reply:
Sorry. I keep forgetting this, and `flake8` is happy to let it pass.